python/paddle/fluid/tests/unittests/test_imperative_optimizer.py

```diff
@@ -60,11 +64,32 @@ def _reader_imple():
 
         return _reader_imple
 
-    def _check_mlp(self):
+    def _check_exception(self, exception_message, place=None):
+        seed = 90
+        batch_size = 128
+        if place == None:
+            place = fluid.CPUPlace() if not core.is_compiled_with_cuda(
```

Comment:
NIT (Not Important) but would like you to know the tips.
Prefer to write positive condition in "if" clause. For example:

place = fluid.CUDAPlace(0) if core.is_compiled_with_cuda() else fluid.CPUPlace()

It's not important in your code, but in some cases Software Engineer (SWE) may make it a mess. 

For example, some SWEs may write bool function "SomethingNotTrue", then using "not" before the function. In extreme situation, it can grow a lot "negative and negative" sentences and usually makes code long and not easy to read.

So we usually make code straight in code practice, just like our work :-) For example, the if-else with positive case. (exception: you have "if not something" without a "else")

Reply:
done, thx.